### ISSUE TITLE
NoMethodError in the job_enabled? method

### ISSUE THREAD

**Issue by reubenbrown** [moove-it/sidekiq-scheduler#195]
Occasionally when trying to enable/disable a recurring job in the Sidekiq UI I get this error (full stacktrace below). Based on the error it appears the Sidekiq module was not loaded? 

What's weird is that when I try to enable/disable a recurring job a second time (after the first failure) everything works as expected.
 
```
gems/sidekiq-scheduler-2.1.5/lib/sidekiq/scheduler.rb:349:in `job_enabled?': undefined method `[]' for nil:NilClass (NoMethodError)
    from gems/sidekiq-scheduler-2.1.5/lib/sidekiq/scheduler.rb:355:in `toggle_job_enabled'
    from gems/sidekiq-scheduler-2.1.5/lib/sidekiq-scheduler/web.rb:25:in `block in registered'
    from gems/sidekiq-5.0.2/lib/sidekiq/web/application.rb:266:in `instance_exec'
    from gems/sidekiq-5.0.2/lib/sidekiq/web/application.rb:266:in `block in call'
    from gems/sidekiq-5.0.2/lib/sidekiq/web/application.rb:262:in `catch'
    from gems/sidekiq-5.0.2/lib/sidekiq/web/application.rb:262:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/xss_header.rb:18:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/base.rb:50:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/base.rb:50:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/path_traversal.rb:16:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/json_csrf.rb:26:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/base.rb:50:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/base.rb:50:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/frame_options.rb:31:in `call'
    from gems/rack-protection-2.0.0/lib/rack/protection/base.rb:50:in `call'
    from gems/rack-2.0.3/lib/rack/session/abstract/id.rb:232:in `context'
    from gems/rack-2.0.3/lib/rack/session/abstract/id.rb:226:in `call'
    from gems/rack-2.0.3/lib/rack/urlmap.rb:68:in `block in call'
    from gems/rack-2.0.3/lib/rack/urlmap.rb:53:in `each'
    from gems/rack-2.0.3/lib/rack/urlmap.rb:53:in `call'
    from gems/rack-2.0.3/lib/rack/builder.rb:153:in `call'
    from gems/sidekiq-5.0.2/lib/sidekiq/web.rb:103:in `call'
    from gems/sidekiq-5.0.2/lib/sidekiq/web.rb:108:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/routing/mapper.rb:17:in `block in <class:Constraints>'
    from gems/actionpack-5.0.3/lib/action_dispatch/routing/mapper.rb:46:in `serve'
    from gems/actionpack-5.0.3/lib/action_dispatch/journey/router.rb:39:in `block in serve'
    from gems/actionpack-5.0.3/lib/action_dispatch/journey/router.rb:26:in `each'
    from gems/actionpack-5.0.3/lib/action_dispatch/journey/router.rb:26:in `serve'
    from gems/actionpack-5.0.3/lib/action_dispatch/routing/route_set.rb:725:in `call'
    from gems/warden-1.2.7/lib/warden/manager.rb:36:in `block in call'
    from gems/warden-1.2.7/lib/warden/manager.rb:35:in `catch'
    from gems/warden-1.2.7/lib/warden/manager.rb:35:in `call'
    from gems/rack-2.0.3/lib/rack/etag.rb:25:in `call'
    from gems/rack-2.0.3/lib/rack/conditional_get.rb:25:in `call'
    from gems/rack-2.0.3/lib/rack/head.rb:12:in `call'
    from gems/rack-2.0.3/lib/rack/session/abstract/id.rb:232:in `context'
    from gems/rack-2.0.3/lib/rack/session/abstract/id.rb:226:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/cookies.rb:613:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
    from gems/activesupport-5.0.3/lib/active_support/callbacks.rb:97:in `__run_callbacks__'
    from gems/activesupport-5.0.3/lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
    from gems/activesupport-5.0.3/lib/active_support/callbacks.rb:90:in `run_callbacks'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/callbacks.rb:36:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
    from gems/railties-5.0.3/lib/rails/rack/logger.rb:36:in `call_app'
    from gems/railties-5.0.3/lib/rails/rack/logger.rb:24:in `block in call'
    from gems/activesupport-5.0.3/lib/active_support/tagged_logging.rb:69:in `block in tagged'
    from gems/activesupport-5.0.3/lib/active_support/tagged_logging.rb:26:in `tagged'
    from gems/activesupport-5.0.3/lib/active_support/tagged_logging.rb:69:in `tagged'
    from gems/railties-5.0.3/lib/rails/rack/logger.rb:24:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/request_id.rb:24:in `call'
    from gems/rack-2.0.3/lib/rack/method_override.rb:22:in `call'
    from gems/rack-2.0.3/lib/rack/runtime.rb:22:in `call'
    from gems/actionpack-5.0.3/lib/action_dispatch/middleware/executor.rb:12:in `call'
    from gems/rack-2.0.3/lib/rack/sendfile.rb:111:in `call'
    from gems/railties-5.0.3/lib/rails/engine.rb:522:in `call'
    from gems/unicorn-5.2.0/lib/unicorn/http_server.rb:562:in `process_client'
    from gems/unicorn-5.2.0/lib/unicorn/http_server.rb:658:in `worker_loop'
    from gems/unicorn-5.2.0/lib/unicorn/http_server.rb:508:in `spawn_missing_workers'
    from gems/unicorn-5.2.0/lib/unicorn/http_server.rb:132:in `start'
    from gems/unicorn-5.2.0/bin/unicorn:126:in `<top (required)>'
    from bin/unicorn:22:in `load'
    from bin/unicorn:22:in `<main>'
```

**Comment by snmgian** [comment]
Pushed PR #196

**Comment by snmgian** [comment]
Released in v2.1.6
